src/examples/sankey/README.md

```diff
@@ -3,7 +3,12 @@
 
 ![](sankey.png)
 
-This diagram creates a [sankey diagram](https://en.wikipedia.org/wiki/Sankey_diagram) to display sequences of transitions.
+This diagram creates a [sankey diagram](https://en.wikipedia.org/wiki/Sankey_diagram) to display sequences of transitions.S ankey diagrams are a specific type of flow diagram, in which the width of the arrows is shown proportionally to the flow quantity. Sankey diagrams are often used in fields of science, especially physics. They are used to represent energy inputs, useful output, and wasted output. Sankeys can also be used to understand how users arrive at and navigate through an application or website. 
```

Comment:
/.S ankey/Sankey

I can't wrap my mind around whether this sentence uses "is" correctly...
Suggested a change, but it could be wrong. Defer to @kenvlooker 
> in which the width of the arrows is shown proportionally to the flow quantity
 "in which the width of the arrows are proportional to the flow of quantity" ?

Reply:
Fixes submitted in a separate PR:

- Fixed "S ankey"
- "in which the width of the arrows is shown proportionally..." ==> "...in which the width of each band is shown proportionally..." 
- "fields of science" ==> "scientific fields"
- **Question:** Would it make sense to edit "This diagram creates a sankey diagram..."? Change to "This code/script/? creates a sankey diagram..."??